src/components/Modals/FiatRamps/views/RampsList.tsx

```diff
@@ -25,17 +25,15 @@ type RampsListProps = {
 
 export const RampsList: React.FC<RampsListProps> = ({ setFiatRampProvider }) => {
   const history = useHistory()
-  const mtPelerinFiatRampFlag = useFeatureFlag('MtPelerinFiatRamp')
+  const onRamperFiatRampFlag = useFeatureFlag('OnRamperFiatRamp')
   const tagColor = useColorModeValue('gray.600', 'gray.400')
   const ramps = useMemo(() => {
     type Entry = [keyof typeof supportedFiatRamps, SupportedFiatRampConfig]
     const initial: ReactElement[] = []
     const result = (Object.entries(supportedFiatRamps) as Entry[]).reduce(
       (acc, supportedFiatRamp) => {
         const [fiatRamp, fiatRampConfig] = supportedFiatRamp
-        // TODO: remove before merging to develop
-        if (fiatRamp === 'MtPelerin' && !mtPelerinFiatRampFlag) return acc
-        if (fiatRampConfig.isImplemented) {
+        if (fiatRampConfig.isImplemented && fiatRampConfig.isActive) {
```

Comment:
Just a proposal so that we don't have to add per-provider FF handling and can keep the code generic. Using the hook isn't an option inside a static config, but I guess it's not too bad if we just load the env var directly and keep the code down here cleaner

Reply:
While I conceptually agree with this addition from a high-level, realized there is one catch here: this will only work with feature flags loaded from env, not with setting it in the flags UI, which overrides env flags with the ones we have in the store.

If we want to have an `isActive` property, we should probably make it a function:
- either a `(flagToCheck: FeatureFlags): boolean` method that would effectively just evaluate the flag you pass to it
- alternatively, a pure function that would call the `selectFeatureFlags` selector with `store.getState()`, which allows us to access the redux store outside of react, as long as we're in the context of the redux provider (which almost all the app is except for a few entrypoints)

Reply:
I went with the first option, gets the job done